PHP81 Support

tertek opened the pull request:
Fixes Type Error that throws Fatal Error in PHP8.1. See https://github.com/susom/mass-delete/issues/10

tertek committed:
PHP81 Support

Fixes Type Error that throws Fatal Error in PHP8.1.
See https://github.com/susom/mass-delete/issues/10

dr01d3r commented:
@tertek @123andy @yasukawa 

Maybe I'm misreading this, but looking at the signature for `Records::deleteRecord` shouldn't this fix just be to correct the project array typo?

`$Proj->project['randomization'],`

tertek commented:
@dr01d3r Thanks for this! You are right: Giving `$Proj->project_id` as randomization parameter does not make sense at all!

The other two changes in use with `count()` are still needed though.

tertek committed:
Apply correction

As commented here:
https://github.com/susom/mass-delete/pull/11#issuecomment-1284604645
